### ISSUE TITLE
Possible typo

### ISSUE THREAD

**Issue by OxMarco** [BennyTheDev/pipe-wallet#7]
https://github.com/BennyTheDev/pipe-wallet/blob/93782aea0601d0ecd0040660acbf641e6cdb7248/pipe.mjs#L1448

Shouldn't this be `amount + fee * 2n` ?

**Comment by OxMarco** [comment]
Here too

https://github.com/BennyTheDev/pipe-wallet/blob/93782aea0601d0ecd0040660acbf641e6cdb7248/pipe.mjs#L1626

**Comment by thewrlck** [comment]
because if found is 2000 and rate is 10 and amount is 1000 so

2000 >= 1000 * 10 * 2n and got error

**Comment by BennyTheDev** [comment]
No, it means that if more or equal is found, it will break the loop and use the unspents that have been collected to this point.
